nym-vpn-core/crates/nym-vpn-lib/src/tunnel_state_machine/tunnel/mixnet/connector.rs

```diff
@@ -72,6 +73,13 @@ impl Connector {
             exit_mix_addresses,
             interface_addresses,
         };
+        if let Some(exit_country_code) = selected_gateways.exit.two_letter_iso_country_code() {
+            self.mixnet_client
+                .send_stats_event(
+                    ConnectionStatsEvent::MixCountry(exit_country_code.to_string()).into(),
```

Comment:
Doesn't this heavily deanonymises the mixnet mode of the vpn, by creating a link between mixnet client's identity and the exit gateway? At least in the current state, where SURBs are not implemented for mixnet mode.

Reply:
So this info gets sent to the aggregator and the report including this is sent through the mixer, with a stats id, hashed from the identity. 
You're indeed reducing the set of gateway you might be connected to, but you're not doxing yourself by revealing your entry gateway